README.md

```diff
@@ -104,10 +104,8 @@ $ kubectl create -f deployments/crdnetwork.yaml
 ```
 $ git clone https://github.com/intel/sriov-cni.git
 $ cd sriov-cni
-$ git fetch
-$ git checkout dev/k8s-deviceid-model
-$ ./build
-$ cp bin/sriov /opt/cni/bin
+$ make
```

Comment:
nit: we may just need to run "make build" to get the 'sriov' cni binary, but it doesn't hurt to run fmt, golint checking before build .
LGTM.

Reply:
Thanks for review! Just noticed this comment after merged :)